### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6085,6 +6088,7 @@ documentataions->documentations
 documentaton->documentation
 documentes->documents
 documention->documentation
+documentserivce->documentservice
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Seems specific.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -6116,6 +6120,7 @@ doesnt->doesn't, does not,
 doesnt;->doesn't
 doess->does
 doestn't->doesn't
+dof->of
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest doff as in "doff your hat" ( https://www.phrases.org.uk/meanings/doff-your-hat.html )

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -6130,6 +6135,7 @@ domians->domains
 dominaton->domination
 dominent->dominant
 dominiant->dominant
+donejun->dungeon
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Plural

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

**Pull request merged**